Makefile

```diff
@@ -1,6 +1,62 @@
 SHELL := /bin/bash
 PYTHON_CODE_DIRECTORIES = src/py
 
+
+##########
+# DATABASE
+##########
+
+LOCAL_DB_CONTAINER_NAME = covidr-local
+LOCAL_DB_CONTAINER_ID = $(shell docker ps -a | grep $(LOCAL_DB_CONTAINER_NAME) | awk '{print $$1}')
+LOCAL_DB_CONTAINER_RUNNING_ID = $(shell docker ps | grep $(LOCAL_DB_CONTAINER_NAME) | awk '{print $$1}')
+LOCAL_DB_NAME = covidr_db
+LOCAL_DB_ADMIN_USERNAME = postgres  # This has to be "postgres" to ease moving snapshots from RDS.
+LOCAL_DB_ADMIN_PASSWORD = admin
+LOCAL_DB_RW_USERNAME = user_rw
+LOCAL_DB_RW_PASSWORD = password_rw
+LOCAL_DB_RO_USERNAME = user_ro
+LOCAL_DB_RO_PASSWORD = password_ro
+DOCKER_IMAGE = czbiohub/covidhub-postgres:11.5-alpine
+
+start-local-db:
+	@if [ "$(LOCAL_DB_CONTAINER_ID)" == "" ]; then \
+		docker create --name $(LOCAL_DB_CONTAINER_NAME) -p 5432:5432 \
+		-e POSTGRES_USER=$(LOCAL_DB_ADMIN_USERNAME) \
+		-e POSTGRES_PASSWORD=$(LOCAL_DB_ADMIN_PASSWORD) \
+		-e POSTGRES_DB=$(LOCAL_DB_NAME) \
+		$(DOCKER_IMAGE); \
+	fi
+	@if [ "$(LOCAL_DB_CONTAINER_RUNNING_ID)" == "" ]; then \
+		docker start $(LOCAL_DB_CONTAINER_NAME) && sleep 3; \
+	fi
+
+setup-local-db:
+	@$(MAKE) start-local-db
+	@docker exec $(LOCAL_DB_CONTAINER_NAME) psql -h localhost -d $(LOCAL_DB_NAME) -U $(LOCAL_DB_ADMIN_USERNAME) -c "CREATE USER $(LOCAL_DB_RW_USERNAME) WITH PASSWORD '$(LOCAL_DB_RW_PASSWORD)';"
+	@docker exec $(LOCAL_DB_CONTAINER_NAME) psql -h localhost -d $(LOCAL_DB_NAME) -U $(LOCAL_DB_ADMIN_USERNAME) -c "CREATE USER $(LOCAL_DB_RO_USERNAME) WITH PASSWORD '$(LOCAL_DB_RO_PASSWORD)';"
+	@docker exec $(LOCAL_DB_CONTAINER_NAME) psql -h localhost -d $(LOCAL_DB_NAME) -U $(LOCAL_DB_ADMIN_USERNAME) -c "GRANT CREATE ON DATABASE $(LOCAL_DB_NAME) TO $(LOCAL_DB_RW_USERNAME);"
+
+init-local-db:
+	@$(MAKE) setup-local-db
+	covidr-cli db create
+	ENV=local alembic stamp head
+
+stop-local-db:
+	@if [ "$(LOCAL_DB_CONTAINER_RUNNING_ID)" != "" ]; then \
+		docker stop $(LOCAL_DB_CONTAINER_NAME); \
+	fi
+
+drop-local-db:
+	@$(MAKE) stop-local-db
+	@( read -p "Delete local database container? [y/N]: " sure && case "$$sure" in [yY]) true;; *) false;; esac )
```

Comment:
it's throwing me off that this @ sign is red, do you know why it's doing that?

Reply:
i looked at the same line in the covidhub Makefile and it looks the same so i think this is correct, i was jw

Reply:
the `@` silences the echoing of the command when it's run.  it's mostly for aesthetic reasons.  Not sure why it's highlighted red.